services/src/test/scala/cromwell/services/metadata/impl/MetadataDatabaseAccessSpec.scala

```diff
@@ -473,9 +473,29 @@ class MetadataDatabaseAccessSpec extends FlatSpec with Matchers with ScalaFuture
       } yield()).futureValue
     }
 
+    it should "properly query metadata summaries based on archived status and timestamp and update archive status after metadata deletion" taggedAs DbmsTest in {
+      (for {
+        workflowId1 <- succeededWorkflowMetadata(WorkflowId.fromString("11111111-1111-1111-1111-111111111112"))
```

Comment:
TOL: Where are these IDs coming from? Could the literals here be converted into references to `val` definitions higher up?

Reply:
Those are just magic numbers from my imagination. I don't think there's any profit in making them `val`s, since we don't use those string literals anywhere else in the test. Also the rest of this test suite is like that - string literal ids are not extracted into `val`s.